src/test/java/hudson/plugins/copyartifact/CopyArtifactTest.java

```diff
@@ -601,6 +608,11 @@ public void testJobRename() throws Exception {
     public void testSavedBuildSelector() throws Exception {
         FreeStyleProject other = createArtifactProject(),
                          p = rule.createFreeStyleProject();
+        ParameterDefinition paramDef = new StringParameterDefinition("FOO","foo");
+        ParametersDefinitionProperty paramsDef = new ParametersDefinitionProperty(paramDef);
+        other.addProperty(paramsDef);
+        CopyArtifactPermissionProperty cpp=new CopyArtifactPermissionProperty("/"+p.getName());
```

Comment:
I believe you don't need `/` as `CopyArtifactPernissionProperty` accepts relative names.
And you'd better to use `getFullName()` if you want to express with absolute names.